src/main/java/com/google/sps/servlets/UserDataServlet.java

```diff
@@ -51,7 +62,19 @@
   private static final String USER_FRIENDS_LIST_PROPERTY = "friends-list";
   private static final String USER_ID_PROPERTY = "id";
   private static final String USER_NAME_PROPERTY = "name";
+  private static final String USER_PROFILE_PHOTO = "profile-photo";
+  private static final String USER_PHOTO_2 = "photo-2";
+  private static final String USER_PHOTO_3 = "photo-3";
+  private static final String USER_PHOTO_4 = "photo-4";
+  private static final String USER_PHOTO_5 = "photo-5";
+  private static final String PROFILE_PHOTO_BLOBKEY = "profile-photo-blobkey";
+  private static final String PHOTO_2_BLOBKEY = "photo-2-blobkey";
+  private static final String PHOTO_3_BLOBKEY = "photo-3-blobkey";
+  private static final String PHOTO_4_BLOBKEY = "photo-4-blobkey";
+  private static final String PHOTO_5_BLOBKEY = "photo-5-blobkey";
 
+  private boolean testing = false;
```

Comment:
Generally, branching behavior for testing is a bad practice as it makes code harder to reason about (tests diverge from production). If there is some functionality that doesn't work in a test environment, prefer to use a mock/fake to override that behavior.